Gemfile

```diff
@@ -23,6 +23,8 @@ gem 'yajl-ruby',       '~> 1.4.0'
 
 gem 'jemalloc',        git: 'https://github.com/joshk/jemalloc-rb'
 
+gem 'irb', require: false
```

Comment:
What does `irb` do? Maybe put it in the commit message so we know why it is here?